app/assets/js/services/elasticsearch.js

```diff
@@ -7,9 +7,25 @@ export const ELASTICSEARCH_PROXY_ENDPOINT = `${host}:${port}/_search`;
 export const ELASTICSEARCH_INDEX_NAME = __ELASTICSEARCH_INDEX__;
 
 const Elasticsearch = require("elasticsearch");
+
```

Comment:
I'd love to get `Elasticsearch` out of the client-side after we push this through, as part of the "Meadow re-evaluate" batch of client-side app work.   As long as this works and queries are firing, even if just limping along or not pretty, we'll extract it soon enough and don't need to spend too much headspace on it.

Reply:
I agree. The newer version of the official JS client even says that [in-browser use is not supported](https://github.com/elastic/elasticsearch-js#browser).